### DISCUSSION TITLE
Blog 6

### DISCUSSION THREAD

**Discussion by Ferreidi** [Anastylos/Jump-and-Run#71] [Blog]
### Fortschritt der Woche

In dieser Woche hat sich Lukas damit befasst, dass die Gegner im Nahkampf sich bewegen können sowie, auch die Effekte der Totems an die Gegner angewendet werden können. Währenddessen hat Michi das Ziel programmiert, damit nun eine Animation erscheint sobald man diese erreicht.

### Einblicke in die Demo

Hier ein paar Einblicke zu unserem Fortschritt:

![2024-11-19_Gegnerbewegung](https://github.com/user-attachments/assets/e69380ca-d0d5-4001-8f9d-3c4b4d12b0b8)

![2024-11-19_Gegner_verbrennen](https://github.com/user-attachments/assets/985e7431-1530-4e2c-ab81-d4742cc0ad87)

![2024-11-19_Gegner_vereisen](https://github.com/user-attachments/assets/d5ce0b2f-1e84-4066-ad4c-eafd1b4bbdfa)

![2024-11-19_Ziel](https://github.com/user-attachments/assets/62a38502-a1c6-4c4f-9dce-3acf64d1b929)

## Aufgabenverteilung und Mitwirkende

- **Game Design & Level Testing:** Don, Dinis, Lukas, Christoph
- **Implementierung der Grundmechaniken:** Michael, Christoph
- **Technische Umsetzung und Fehlerbehebung:** Michael, Christoph

**Comment by SidMllr** [comment]
Euer Fortschritt klingt beeindruckend! Besonders die Implementierung der Totem-Effekte und der Nahkampfbewergung der Gegner sind spannende Ergänzungen. Auch die Animation beim Erreichen des Ziels sorgt für ein verbessertes Spielerlebnis und sieht sehr gut aus. Großartig, wie klar die Aufgaben im Team verteilt sind – das zeigt eine starke Zusammenarbeit.
Eine ganz allgemeine Frage hätten wir noch: Plant ihr, das Spiel zu veröffentlichen, wenn es fertig ist? Falls ja, wie genau, also auf welcher Plattform?
Euer Project Haelth-Team

**Comment by oskarkreutzer** [comment]
Toller Fortschritt! Die neuen Animationen und Effekte bringen das Projekt sichtbar voran, und es ist beeindruckend, wie strukturiert ihr die Aufgaben verteilt habt. Besonders die Animationen helfen, die Entwicklung nachzuvollziehen. Wir ziehen unseren Hut!

Das GymTrackerX Team
